plugin/pkg/scheduler/factory/factory.go

```diff
@@ -471,23 +471,6 @@ func (f *ConfigFactory) ResponsibleForPod(pod *v1.Pod) bool {
 
 func getNodeConditionPredicate() corelisters.NodeConditionPredicate {
 	return func(node *v1.Node) bool {
-		for i := range node.Status.Conditions {
```

Comment:
This is turning on the beta feature which substitutes very important GA one. We need to have a flag gate to allow users to disable this change, if it proves problematic.